.github/workflows/deno.yml

```diff
@@ -0,0 +1,42 @@
+name: Deno
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  denofmt-and-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: denoland/setup-deno@v1
+        with:
+          deno-version: v1.x
+      - uses: actions/checkout@v3
+
+      - run: deno lint
+      - run: deno fmt --check
+
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: denoland/setup-deno@v1
+        with:
+          deno-version: v1.x
+      - uses: actions/checkout@v3
+
+      - run: deno cache source/*.ts
+      - run: deno task check
+      - run: deno test
+
+  backport:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install Dependencies
+        run: npm install --ignore-scripts
+
+      - name: Backport
+        run: npm run backport
+
+      - run: npm pack
```

Comment:
Redundant: `npm pack` invokes `"prepare"` script.

Reply:
The idea is to get the output what is included in the package. Does the packing works? If not the publish will not work anyway. So I like this as a CI step.